v2/samples/servicebus/v1api/v1api20210101preview_namespacesauthorizationrule.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: servicebus.azure.com/v1api20210101preview
+kind: Namespace
+metadata:
+  name: aso-namespace
+  namespace: default
+spec:
+  location: westcentralus
+  owner:
+    name: aso-sample-rg
+  sku:
+    name: Standard
+  zoneRedundant: false
+---
+apiVersion: servicebus.azure.com/v1api20210101preview
```

Comment:
Normally we don't have two resources in a single sample resource. Can you split this into two and just use the existing `namespace`? That's what most samples usually do.

Reply:
I don't understand how the samples are now set up, the whole area suddenly became extremely complex. 
I thought each sample file needed to be standalone. Have removed the extra.